src/commands/app/delete/event.js

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseCommand = require('../../../BaseCommand')
+const yeoman = require('yeoman-environment')
+const debug = require('debug')('aio-cli-plugin-app:init')
+const { flags } = require('@oclif/command')
+
+class DeleteEventCommand extends BaseCommand {
+  async run () {
+    const { args, flags } = this.parse(DeleteEventCommand)
+
+    debug('deleting an action from the project, with args', args, 'and flags:', flags)
```

Comment:
here when you will switch to aioLogger be careful as I don't think it supports multi args like debug() does

also `deleting an events action..`